Resources/Audio/_Cats/StationEvents/attributions.yml

```
@@ -1,3 +1,7 @@
 - files: ["spawn.ogg"]
   license: "CC-BY-NC-ND-4.0"
-  copyright: "Owner boich9585 (discord) and head developer mynicknamewastaken (discord) of Space Station 14 server Dead Space"
\ No newline at end of file
+  copyright: "Owner boich9585 (discord) and head developer mynicknamewastaken (discord) of Space Station 14 server Dead Space"
+
+- files: ["announce_dig.ogg"]
+  license: "CC-BY-NC-ND-4.0"
+  copyright: "https://github.com/EndrAnimet/DeltaV-Stray/commit/7d87605ae4ceb96d9050b8d0d3717fc56c63b3a9#diff-0eb5df8d137cd4711bbde50acf9057a12a5c1efbb03badecabbb635af4d9fcb5R75"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)